Gems/Atom/RHI/Vulkan/Code/Source/RHI/Instance.h

```diff
@@ -47,13 +48,22 @@ namespace AZ
             RHI::PhysicalDeviceList GetSupportedDevices() const;
             AZ::RHI::ValidationMode GetValidationMode() const;
 
+            //! Since the native instance is created when the RHI::Vulkan module was initialized
+            //! this method allows us to re-update it if XR is support is requested
+            void UpdateNativeInstance(RHI::XRRenderingInterface* xrSystem);
+
         private:
             RHI::PhysicalDeviceList EnumerateSupportedDevices() const;
+            void ShutdownInstance();
```

Comment:
This is a bit odd to me. Because this class has a public shut down call and itself is named Instance.
Maybe using a different name like ShutdownVkInstance or ResetInstance? Or leaving a comment to explain what it does.